hack/config.sh

```diff
@@ -1,5 +1,5 @@
 binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer"
+docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirtd"
```

Comment:
The following packages are explicitly added for KubeVirt: util-linux, libcgroup-tools, ethtool, sudo, and docker. Unless we add them to the upstream generic libvirt image (which seems wrong), then a call to yum/dnf is unavoidable. That said, @fabiand's observation might be the right answer.